### PR TITLE
Prepare course for release

### PR DESCRIPTION
**Pull request by mattdavis0351**
This PR addresses the release checklist for this course.  The [current progress](https://github.com/github/learning-lab-content/issues/34) can be found in the tracking issue.

### CODE REVIEW

**Comment by mattdavis0351** [comment]
image

File: config.yml
Comment (hectorsector) [inline config.yml:156]: I think it's more meaningful for the learner to trigger the `issue_comment`, what do you think @mattdavis0351?

File: config.yml
Comment (hectorsector) [inline config.yml:274]: This is another case where I think it makes a lot more sense for the learner to trigger the response than for the bot. The way I see it, it kind of takes all the fun out of creating my own workflow and scripts if a bot is causing all the fun stuff to happen instead of me!

File: config.yml
Comment (mattdavis0351) [reply config.yml:156]: I'm not a fan of it.  The course isn't "can the learner open an issue" if that makes sense.  Its focus is on GitHub Script and this is not much different than us opening Pulls for the learner when we need to.

Also, this workflow wont trigger until it is merged.  That's just the nature of using issues as a trigger.  So we would have to somehow tell the learner to merge the workflow, then instruct them to open an issue.

That creates a gap where they can stall the course.  Where do we instruct them to open the issue AFTER the pull is merged, without opening one ourselves?

File: config.yml
Comment (mattdavis0351) [reply config.yml:274]: at the end of the course the learner opens their own issue to demonstrate not supplying a tag. again I echo my feelings about issues not being the focal point of the course.

Arguably this was to demonstrate a potential use-case for automation and GitHub Script.  In that spirit it would be other contributors opening issues in a projects repo, which is simulated by us opening the issue during this course.

Comment (hectorsector) [reply config.yml:274]: I hear ya, but as a learner that took your course for the first time, I didn't really understand what was going on when things were just automatically happening. This wasn't a good learning experience for me. I ultimately leave it up to you, which is why I left these notes as comments and not direct commits, but that's my feedback!

Comment (mattdavis0351) [reply config.yml:274]: I appreciate your feedback!  I'm gonna see if I can get some more opinions on this before deciding one way or the next.

**Comment by brianamarie** [comment]
I'm going through this course for the 2nd time with an eye on the flow with the bot activating the triggers for the actions. It didn't stand out to me before, but I do think there's a chance to reduce confusion here. If I'm understanding correctly, I see the pros/cons as:

#### The bot opening the issue

- 👍One less step for the learner to complete 
- 👍All learner actions are about writing code
- 👎We are assuming that the learner can differentiate between bot activity and action-triggered activity (This is a big one to me - there is a lot of mystery behind Learning Lab, and I think it's a risk to let users confuse the Learning Lab things and the Action things. We can't validate their understanding any other way.)

#### The Learner opening the issue

- 👍Learner has to mentally connect the dots that the action is triggered by them opening an issue, or commenting, or whatever was specified in the workflow 

I don't see the learner opening issues as a practice in opening issues for no reason - I see it as solidifying the understanding of the workflow that we've asked them to build. 

All that said, it didn't occur to me when I went through the course for the first time. Just my two cents 💰

**Comment by mattdavis0351** [comment]
@brianamarie imagine a real world project.  I set up the workflow for that repo, and that workflow responds to issues when they are opened by the community.  Once I did that I would have to wait for people to open issues themselves, aside from the ones that I may or may not open to track things.

By having the bot open the issue, I feel as though it simulates a more accurate representation of this workflow in the wild.  The bot essentially becomes a community member that has stumbled on your repo and opened an issue for whatever reason.

I worry that if the learner is the one who triggers this workflow then we lose that simulation.  We could also run the risk of the learner assuming this workflow is tied to them opening issues, and not anyone in the world.  That risk is small, but if I'm super new to GitHub and collaboration this is a potential misconception.

Taking the simulated aspect into account, does that change your feedback in anyway?

**Comment by brianamarie** [comment]
@mattdavis0351 I completely see what you mean. My reasoning is that while simulating the environment is important, it may lose value as a training exercise if it's so realistic that we sacrifice the scaffolding that creates a positive learning experience. 

As a possible compromise, what if we had the user trigger the first several instances to solidify the knowledge, then show it as another collaborator later in the course?

**Comment by hectorsector** [comment]
@mattdavis0351 I added some validations that I think will help!

**Review by hectorsector — APPROVED** [review]
@mattdavis0351 once we merge this branch I'll get our team testing. We can open new PRs so it isn't the end!

**Comment by hectorsector** [comment]
@mattdavis0351 based on our conversations over Slack, I'm going to merge this so I can get this course QA'd.

**Pull request merged**
